setup.cfg

```diff
@@ -2,7 +2,47 @@
 universal = 1
 
 [metadata]
+name = python-dateutil
+description = Extensions to the standard Python datetime module
+author = Gustavo Niemeyer
+author_email = gustavo@niemeyer.net
+maintainer = Paul Ganssle
+maintainer_email = dateutil@python.org
+url = https://dateutil.readthedocs.io
+long_description = file: README.rst
```

Comment:
You should revert this change, because we're modifying the `README` variable in `setup.cfg`.

Reply:
Thank you, problem solved!